### PR TITLE
Issue #2572119: Replace evaluate() with doEvaluate() in EntityIsNew c…

### PR DESCRIPTION
**Pull request by emclaughlin1215**
…ondition.

See d.o issue https://www.drupal.org/node/2572119

### CODE REVIEW

File: src/Plugin/Condition/EntityIsNew.php [hunk 1]
```diff
@@ -29,10 +30,15 @@
...
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $provided_entity
+   *   The entity to check if is new.
+   *
+   * @return bool
+   *   TRUE if provided entity is new.
    */
-  public function evaluate() {
-    $provided_entity = $this->getContextValue('entity');
+  protected function doEvaluate(EntityInterface $provided_entity) {
```
Comment (klausi) [inline src/Plugin/Condition/EntityIsNew.php]: Let's use $entity as name here to match the name in the annotation.

File: src/Plugin/Condition/EntityIsNew.php [hunk 2]
```diff
@@ -29,10 +30,15 @@
 class EntityIsNew extends RulesConditionBase {
 
   /**
-   * {@inheritdoc}
+   * Check if the provided entity is new.
+   *
+   * @param \Drupal\Core\Entity\EntityInterface $provided_entity
+   *   The entity to check if is new.
```
Comment (klausi) [inline src/Plugin/Condition/EntityIsNew.php]: "The entity to check." should be enough.

File: src/Plugin/Condition/EntityIsNew.php [hunk 3]
```diff
@@ -29,10 +30,15 @@
...
+   *
+   * @return bool
+   *   TRUE if provided entity is new.
```
Comment (klausi) [inline src/Plugin/Condition/EntityIsNew.php]: "... if the provided entity ..."

**Comment by emclaughlin1215** [comment]
Addressed the comments and pushed up changes.

**Comment by klausi** [comment]
merged, thanks!

**Pull request closed without merging**